main.py

```diff
@@ -0,0 +1,16 @@
+import argparse
+from source import brightnessChange
+
+def main():
+    parser = argparse.ArgumentParser(description='Change lightness of a provided image.')
+    parser.add_argument("--filepath", type=str, default='sky.jpg', help='Provide the input path of the image.')
+    parser.add_argument("--lightness", type=int, default=10, help='Input the range of lightness (0 to 255).')
```

Comment:
Lightness value is not related to RGB 0 - 255 value. Must be in percentage 0 - 100